simulators/seissol/v.0.2/Dockerfile

```diff
@@ -0,0 +1,206 @@
+    cp metis/include/metis.h ${SEISSOL_PREFIX}/include && \
+    cd .. && rm -rf parmetis-4.0.3*
+
+# Install yaml-cpp
+RUN git clone https://github.com/jbeder/yaml-cpp.git /yaml-cpp && \
+    cd /yaml-cpp && \
+    mkdir build && cd build && \
+    cmake .. -DYAML_BUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=${SEISSOL_PREFIX} && \
+    make -j$(nproc) && make install && \
+    cd ../.. && rm -rf /yaml-cpp
+
+# Install HDF5 with MPI support
+RUN wget https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.12/hdf5-1.12.3/src/hdf5-1.12.3.tar.bz2 && \
+    tar -xaf hdf5-1.12.3.tar.bz2 && \
+    cd hdf5-1.12.3 && \
+    CPPFLAGS="-fPIC ${CPPFLAGS}" \
+    CC=mpicc CXX=mpicxx \
+    ./configure \
+        --enable-parallel \
+        --prefix=/usr/local \
+        --with-zlib \
+        --enable-shared \
+        --enable-static \
+        --with-mpi=/usr/lib/x86_64-linux-gnu/openmpi \
+        --includedir=/usr/include/hdf5/parallel && \
+        make -j$(nproc) && make install && \
+    cd .. && rm -rf hdf5-1.12.3*
+
+# Install ImpalaJIT
+RUN git clone https://github.com/uphoffc/ImpalaJIT.git /ImpalaJIT && \
+    cd /ImpalaJIT && \
+    mkdir build && cd build && \
+    cmake .. -DCMAKE_INSTALL_PREFIX=${SEISSOL_PREFIX} && \
+    make -j$(nproc) && make install && \
+    cd ../.. && rm -rf /ImpalaJIT
+
+# Install NetCDF with Parallel I/O
+RUN wget https://downloads.unidata.ucar.edu/netcdf-c/4.8.1/netcdf-c-4.8.1.tar.gz && \
+    tar -xaf netcdf-c-4.8.1.tar.gz && \
+    cd netcdf-c-4.8.1 && \
+    CFLAGS="-fPIC ${CFLAGS}" CC=h5pcc ./configure --enable-shared=no --prefix=${SEISSOL_PREFIX} --disable-dap && \
+    make -j$(nproc) && make install && \
+    cd .. && rm -rf netcdf-c-4.8.1*
+
+# Install libxsmm gemm generator
+RUN git clone --depth=1 --branch 1.17 https://github.com/hfp/libxsmm && \
+    cd libxsmm && \
+    make generator && \
+    cp bin/libxsmm_gemm_generator ${SEISSOL_PREFIX}/bin/ && \
+    cd .. && rm -rf libxsmm
+
+
+# Install ASAGI
+RUN git clone https://github.com/TUM-I5/ASAGI.git /asagi && \
+    cd /asagi && \
+    git submodule update --init && \
+    mkdir build && cd build && \
+    cmake .. ${CMAKE_COMMON_FLAGS}  \
+    -DSHARED_LIB=ON \
+    -DSTATIC_LIB=ON \
+    -DUSE_MPI=ON \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX=${SEISSOL_PREFIX} \
+    -DCMAKE_INCLUDE_PATH=${SEISSOL_PREFIX}/include \
+    -DCMAKE_LIBRARY_PATH=${SEISSOL_PREFIX}/lib && \
+        make -j$(nproc) && make install && \
+        cd ../.. && rm -rf /asagi
+
+
+# Install EASI
+RUN git clone --recursive https://github.com/SeisSol/easi.git /easi && \
+    cd /easi && \
+    mkdir build && cd build && \
+    cmake .. ${CMAKE_COMMON_FLAGS} -DCMAKE_PREFIX_PATH=${SEISSOL_PREFIX} -DASAGI=ON -DIMPALAJIT=ON -DLUA=ON \
+    -DUSE_MPI=ON && \
+    make -j$(nproc) && make install && \
+    cd ../.. && rm -rf /easi
+
+
+
+# Clone SeisSol with submodules
+RUN git clone --recursive https://github.com/SeisSol/SeisSol.git /SeisSol
```

Comment:
Here we should do one of two things:
- Go to a specific tag/commit
- Download an official release if we have one

The way you have it now we always clone to the latest commit. This could result on two different builds producing two different (versions) of the simulator. We want this step to build the same simulator today and a day from now. 
In this case, you can download the code from the official releases https://github.com/SeisSol/SeisSol/archive/refs/tags/v1.3.0.zip (don't forget to update the folder v0.2 to v1.3.0, this folder has the version of the simulator)

Reply:
two avoid different build for seissol simulator I have updated the code to use a specific commit .

Reply:
Now you just need to update the version folder. If you are using the most recent commit i believe you should be on version 1.3.0, since that is the latest release.

As for the 1.0.1 folder. Since you are using the image seissol/training i'm not sure what version that is.